images/win/Windows2019-Azure.json

```diff
@@ -143,49 +143,27 @@
         {
             "type": "powershell",
             "scripts":[
-                "{{ template_dir }}/scripts/Installers/Update-DotnetTLS.ps1"
-            ]
-        },
-        {
-            "type": "windows-restart",
-            "restart_timeout": "10m"
-        },
-        {
-            "type": "powershell",
-            "scripts":[
+                "{{ template_dir }}/scripts/Installers/Update-DotnetTLS.ps1",
                 "{{ template_dir }}/scripts/Installers/Install-ContainersFeature.ps1"
             ]
         },
-        {
-            "type": "windows-restart",
-            "restart_timeout": "10m"
-        },
         {
             "type": "powershell",
             "inline": [
               "setx ImageVersion {{user `image_version` }} /m",
-              "setx ImageOS {{user `image_os` }} /m"
-            ]
-        },
-        {
-            "type": "powershell",
-            "environment_vars": [
-                "IMAGE_VERSION={{user `image_version`}}",
-                "IMAGEDATA_FILE={{user `imagedata_file`}}"
-            ],
-            "scripts":[
-                "{{ template_dir }}/scripts/Installers/Update-ImageData.ps1"
+              "setx ImageOS {{user `image_os` }} /m",
+              "setx ImageFile {{user `imagedata_file`}} /m"
```

Comment:
Previously, we didn't set this environments variable (we set it only for step, not the whole VM).
Do we really need to set it for whole machine?

Reply:
Thank you, I have changed this back.